x/data/server/testsuite/suite.go

```diff
@@ -43,105 +42,105 @@ func (s *IntegrationTestSuite) TearDownSuite() {
 }
 
 func (s *IntegrationTestSuite) TestScenario() {
-	//testContent := []byte("xyzabc123")
-	//mh, err := multihash.Sum(testContent, multihash.SHA2_256, -1)
-	//s.Require().NoError(err)
-	//cid := gocid.NewCidV1(gocid.Raw, mh)
+	// testContent := []byte("xyzabc123")
```

Comment:
is it still useful to keep commented code committed on master?

Reply:
I don't really know about this 
Do you have any idea on this @blushi ?

Reply:
this is related to the data module which is still WIP so let's keep the comments for now